pandas/tests/indexes/multi/test_constructor.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+import re
+
+import numpy as np
+
+from pandas import (Index, MultiIndex)
+
+import pandas.util.testing as tm
+
+from pandas.tests.indexes.common import Base
+
+
+class TestConstructor(Base):
+    _holder = MultiIndex
+    _compat_props = ['shape', 'ndim', 'size', 'itemsize']
```

Comment:
so things that are not used in a particular module need not be repeated here (e.g. in the constructor) and/or properties in the class.

We want to actually remove the need for the class entirely. You *could* do that here as well (or in a next pass PR).
This is a little trickier because need to turn things (like self.index) into fixtures. so ok with doing this later.